collections/object/shallow-copy-an-object.md

```diff
@@ -1,3 +1,6 @@
 ~~~ javascript
 const shallowCopy = obj => Object.assign({}, obj);
+
+// or
+const shallowCopy = obj => [...obj];
```

Comment:
I think it should be `{...obj}`?

Reply:
Yes my bad, I update the case ! By the way the same method works too in order to clone an array